src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+use Page\Acceptance\Administrator\UserNotesListPage;
+use Page\Acceptance\Administrator\UserNotesFormPage;
+
+/**
+ * Administrator UserNotes Tests
+ *
+ * @category  Users
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class UserNotesCest
+{
+	public function __construct()
+	{
+		$this->username = "admin";
+		$this->password = "admin";
+		$this->name = "Super User";
+		$this->category = "Uncategorised";
+		$this->subject = "Test Subject";
+		$this->editorText = 'This is a test note for user ' . $this->username;
+	}
+
+	/**
+	 * Create a user notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 *
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function createUserNote (\AcceptanceTester $I)
+	{
+		$I->wantToTest('creating user notes ');
+		$I->doAdministratorLogin();
+		$I->amOnPage(UserNotesListPage::$url);
+		// New
+		$I->clickToolbarButton('New');
+		$I->fillField(UserNotesFormPage::$subject,$this->subject);
+		// Select User
+		$I->click(UserNotesFormPage::$selectUserButton);
+		$I->switchToIFrame('Select User');
+		$I->searchForItem($this->username);
+		$I->click(['link' => $this->name]);
+		$I->switchToPreviousTab();
+		// Select category
+		$I->click(UserNotesFormPage::$selectCategory);
+		$I->switchToIFrame('Select or Change Category');
+		$I->searchForItem($this->category);
+		$I->click(['link' => $this->category]);
+		$I->switchToPreviousTab();
+		// Fill editor
+		$I->scrollTo(UserNotesFormPage::$editor);
+		$I->scrollTo(UserNotesFormPage::$toggleEditor);
+		$I->click(UserNotesFormPage::$toggleEditor);
+		$I->fillField(UserNotesFormPage::$editor, $this->editorText);
+		// Save and close
+		$I->click(UserNotesFormPage::$dropDownToggle);
+		$I->clickToolbarButton('save & close');
+		$I->searchForItem($this->subject);
+		$I->see($this->subject);
+	}
+
+	/**
+	 * Unpublish User Notes
+	 *
+	 * @param   \AcceptanceTester  $I The AcceptanceTester Object
+	 * @depends createUserNote
+	 * @since   4.0.0
+	 *
+	 * @return  void
+	 */
+	public function unpublishUserNote (\AcceptanceTester $I)
```

Comment:
Missing the @depends annotation that we talked about

Reply:
@depends is mentioned, see here: https://github.com/joomla/test-system/pull/41/files/43c09b80aafe9448b77398f4170899c34dc3d493#diff-aa63769d16d496c60b7d7c19a78b9caeR77